src/Graby.php

```diff
@@ -70,12 +72,25 @@ public function __construct($config = [], Client $client = null, ConfigBuilder $
 
         $this->config = $resolver->resolve($config);
 
-        $this->debug = (bool) $this->config['debug'];
-        $this->logger = new NullLogger();
+        $this->debug = $this->config['debug'];
+        $this->logLevel = $this->config['log_level'];
 
+        // Debug mode can be activated with 'debug' => true
+        // More details on the retrieved code and its consequent modifications
+        // can be obtained using 'log_level' = 'debug'
         if ($this->debug) {
             $this->logger = new Logger('graby');
-            $this->logger->pushHandler(new StreamHandler(__DIR__ . '/../log/graby.log'));
+
+            if ($this->logLevel == 'debug') {
+                // Emptying of the HTML logfile to avoid gigantic logs
+                // This statement has to be before Logger::INFO to catch all DEBUG messages
+                fclose(fopen(__DIR__ . '/../log/html.log','w'));
+                $this->logger->pushHandler(new StreamHandler(__DIR__ . '/../log/html.log', Logger::DEBUG));
+            }
+
+            $this->logger->pushHandler(new StreamHandler(__DIR__ . '/../log/graby.log', Logger::INFO, false));
+        } else {
+            $this->logger = new NullLogger();
```

Comment:
You can avoid the `else` by letting the definition before the `if` (like it was done before)